server/graphql/v1/mutations/collectives.js

```diff
@@ -463,7 +464,19 @@ export async function archiveCollective(_, args, req) {
   }
 
   // TODO: cascade deactivation to EVENTs and PROJECTs?
+  const subProjects = await collective.getProjects();
```

Comment:
We generally don't use "sub", events and projects are always under a collective.

```suggestion
  const projects = await collective.getProjects();
```